### DISCUSSION TITLE
分秒帧可以创建相同名称的文件夹导致出现一系列错误

### DISCUSSION THREAD

**Discussion by cryingcheems** [AlistGo/alist#368] [Q&A]
我使用rclone挂载webdav（分秒帧）时，上传多个文件到同一目录，会导致出现多个（取决于上传文件个数）相同的文件夹，文件夹里都有一份文件，导致目录混乱，在网页端查看时，虽显示有多个文件夹，但点进去后的内容都是最新一个文件夹中的内容

**Comment by xhofe** [comment]
上传文件多了文件夹？都叫什么名字？

**Reply by cryingcheems** [reply]
上传目录
![image](https://user-images.githubusercontent.com/89782316/149648101-d09a6c2c-0102-487c-a234-b135a131f287.png)

执行rclone copy /root/123 alist:/分秒帧/123

结果变成这样
![image](https://user-images.githubusercontent.com/89782316/149648131-85c02995-0073-4684-a282-bcb3e5ee3538.png)

**Reply by xhofe** [reply]
已禁止新建同名文件夹。
